api4/team.go

```diff
@@ -432,11 +452,32 @@ func addTeamMembers(c *Context, w http.ResponseWriter, r *http.Request) {
 	var err *model.AppError
 	members := model.TeamMembersFromJson(r.Body)
 
-	if len(members) > MAX_ADD_MEMBERS_BATCH || len(members) == 0 {
+	if len(members) > MAX_ADD_MEMBERS_BATCH {
 		c.SetInvalidParam("too many members in batch")
 		return
 	}
 
+	if len(members) == 0 {
```

Comment:
Unrelated to this story; just noticed the validation error didn’t make for the `||` case.